Automattic/sensei
# Add notification area for JITMs on Sensei Home

Fixes #5674

### Changes proposed in this Pull Request

* Create notification area for JITMs on Sensei Home;
* Fix some issues with the notifications backend regarding broken URLs;
* Also create new notices component based on the `Sensei_Admin_Notices` code;

### Testing instructions

On a WP installation containing Sensei LMS based on this branch and Sensei Pro on the New Onboarding Wizard branch - with a license configured:

1. Visit Sensei Home;
2. Verify that no notices appear;
3. Checkout an older Sensei Pro version, let's say, 1.6.2;
4. Verify if a notice appears;
5. Verify what happens when you click on the "What's new" link (it should open the changelog page);
6. Disable Sensei Pro in the plugins list;
7. Verify if another notice appears;
8. Verify if, when clicking on the "Activate" link, it properly activates Sensei Pro;
9. Now, visit senseilms.com and disconnect the Sensei Pro license from there;
10. Verify if the notice changes once again;

Also verify if the styles are correct according to the design specified.
